app/src/main/java/zapsolutions/zap/util/ChannelUtil.java

```diff
@@ -0,0 +1,20 @@
+package zapsolutions.zap.util;
+
+import com.google.protobuf.ByteString;
+import com.google.protobuf.MessageLite;
+
+import java.io.IOException;
+
+public class ChannelUtil {
+
+    public static ByteString serialize(MessageLite channel) {
```

Comment:
Would it make sense to abstract this even more? I can see we will use the serialization in other cases as well in the future. For example for transaction details.
That's why I think we could do something like "SerializeUtil". Or maybe you have a better name for it. 
In that case we should also change the channel variable to message.
` public static ByteString serialize(MessageLite message)`